### ISSUE TITLE
cannot use JSX fragments

### ISSUE THREAD

**Issue by toger5** [bitboxer/jsx-no-react#102]
using 
```
return <>
<div></div>
<div></div>
</>
```
raises the follwoing error:
`Uncaught ReferenceError: React is not defined`
I guess <></> gets translated to: <React.Fragment></React.Fragment>

**Comment by bitboxer** [comment]
Yeah, `Fragment` is not supported yet. If you can create a PR for this, I would be happy to merge it ASAP.
